tests/suites/test_suite_psa_crypto.function

```diff
@@ -1540,15 +1545,28 @@ void hash_bad_order( )
 
     /* Call setup twice in a row. */
     PSA_ASSERT( psa_hash_setup( &operation, alg ) );
+    ASSERT_OPERATION_IS_ACTIVE( operation );
     TEST_EQUAL( psa_hash_setup( &operation, alg ),
                 PSA_ERROR_BAD_STATE );
+    ASSERT_OPERATION_IS_INACTIVE( operation );
     PSA_ASSERT( psa_hash_abort( &operation ) );
+    ASSERT_OPERATION_IS_INACTIVE( operation );
 
     /* Call update without calling setup beforehand. */
     TEST_EQUAL( psa_hash_update( &operation, input, sizeof( input ) ),
                 PSA_ERROR_BAD_STATE );
     PSA_ASSERT( psa_hash_abort( &operation ) );
 
+    /* Check that update calls abort on error. */
+    PSA_ASSERT( psa_hash_setup( &operation, alg ) );
+    operation.ctx.mbedtls_ctx.alg = PSA_ALG_XTS;
```

Comment:
I am not sure what you are doing here and that is not working in the case of a driver.

Reply:
It's to make the following call to psa_hash_update fail on a non-aborted operation, so that I can test that abort has been called inside psa_hash_update as a result of the failure. I'm not sure how else I could test this. In what way will it not work for a driver?

Reply:
`mbedtls_ctx` is the builtin implementation context thus in a configuration where the builtin hash implementation is not included (not possible yet I think though) this may not even compile. I see what you want to do and I can see that making psa_hash_update() fail while passing an active operation context is not that easy. Changing `operation.id` to a non-zero invalid value (big one) would also do the job and be compatible with driver implementation: that the switch in psa_crypto_driver_wrappers.c that would "fail" in that case.